website/screens/components/table/specs/TableSpecsPage.tsx

```diff
@@ -0,0 +1,543 @@
+              <th>Element</th>
+              <th>Core token</th>
+              <th>Value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <code>dataBackgroundColor</code>
+              </td>
+              <td>Cell</td>
+              <td>
+                <code>color-white</code>
+              </td>
+              <td>#ffffff</td>
+            </tr>
+            <tr>
+              <td>
+                <code>dataFontColor</code>
+              </td>
+              <td>Cell value</td>
+              <td>
+                <code>color-black</code>
+              </td>
+              <td>#000000</td>
+            </tr>
+            <tr>
+              <td>
+                <code>headerBackgroundColor</code>
+              </td>
+              <td>Header</td>
+              <td>
+                <code>color-purple-700</code>
+              </td>
+              <td>#5f249f</td>
+            </tr>
+            <tr>
+              <td>
+                <code>headerFontColor</code>
+              </td>
+              <td>Header title</td>
+              <td>
+                <code>color-white</code>
+              </td>
+              <td>#ffffff</td>
+            </tr>
+            <tr>
+              <td>
+                <code>scrollBarThumbColor</code>
+              </td>
+              <td>Scroll bar</td>
+              <td>
+                <code>color-grey-700</code>
+              </td>
+              <td>#666666</td>
+            </tr>
+            <tr>
+              <td>
+                <code>scrollBarTrackColor</code>
+              </td>
+              <td>Scroll bar</td>
+              <td>
+                <code>color-grey-300</code>
+              </td>
+              <td>#cccccc</td>
+            </tr>
+            <tr>
+              <td>
+                <code>sortIconColor</code>
+              </td>
+              <td>Sort</td>
+              <td>
+                <code>color-white</code>
+              </td>
+              <td>#ffffff</td>
+            </tr>
+            <tr>
+              <td>
+                <code>rowSeparatorColor</code>
+              </td>
+              <td>Divider</td>
+              <td>
+                <code>color-grey-300</code>
+              </td>
+              <td>#cccccc</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+        <HeadingLink level={4}>Typography</HeadingLink>
+        <DxcTable>
+          <thead>
+            <tr>
+              <th>Component token</th>
+              <th>Element</th>
+              <th>Core token</th>
+              <th>Value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <code>dataFontFamily</code>
+              </td>
+              <td>Cell value</td>
+              <td>
+                <code>font-family-sans</code>
+              </td>
+              <td>&#39;Open Sans&#39;, sans-serif</td>
```

Comment:
Should we change these codes to actual apostrophes?

Reply:
Done.